express-backend/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We should have license file at repo root folder

Reply:
yea, the license is in root folder. can delete this file.

Reply:
ok